wallet/src/account/mod.rs

```diff
@@ -1270,8 +1298,12 @@ impl Account {
 
         Ok((
             txo.clone(),
-            get_tx_output_destination(txo, &|pool_id| self.output_cache.pool_data(*pool_id).ok())
-                .ok_or(WalletError::InputCannotBeSpent(txo.clone()))?,
+            get_tx_output_destination(
+                txo,
+                &|pool_id| self.output_cache.pool_data(*pool_id).ok(),
+                HtlcSpendingCondition::Undefined,
```

Comment:
`Undefined` looks cryptic. It's better to use a name that would clearly indicate that HTLC outputs won't be included in the result. E.g. something like `SkipHtlc`?
(not insisting on any particular name)

Reply:
Ok I'll rename to `HtlcSpendingCondition::Skip`